### DISCUSSION TITLE
PARALLEL COMPUTING OPENRADIOSS

### DISCUSSION THREAD

**Discussion by CinekPoland** [OpenRadioss/OpenRadioss#930] [Q&A]
I installed Openradioss with inteloneapi base toolkit, inteloneapi hpc toolkit and microsoft mpi.  I have one computer which contain two physical processors Xeon(R) CPU E5-2683 v4  each of this two physical processors have 16 cores. What I have to write in command prompt (CMD) to maximize the power of both of these processors ? 

The order of commands that are entered in CMD is as follows:

1) cd C:\Program Files\OpenRadioss\exec

2) commands.bat

3) starter_win64.exe -i C:\Users\cinek\Desktop\Test_Case_Radioss\cylinder_shear_TN_0000.rad -np 1 -outfile=".\res"

4) engine_win64.exe -i C:\Users\cinek\Desktop\Test_Case_Radioss\cylinder_shear_TN_0001.rad -outfile=".\res"

So second my question is in which command is in wchich command I have write to to maximize the power of both of these processors ?

**Comment by sebastienVilleneuve** [comment]
Hi @CinekPoland 

Good guess is -np 32 in the starter command line.
But you need to use impi binary to launch the engine

( you will find useful information in https://github.com/OpenRadioss/OpenRadioss/blob/main/INSTALL.md )

Thanks,
Sébastien
